releasenotes/notes/fix-fargate-metadata-not-disabled-93e50a3f9fc02240.yaml

```
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fixes a bug where the ECS metadata endpoint V2 would get queried even though it was not configured
+    in the cloud_metadata providers.
```

let's mention the config option name in full, so it's easier to understand the changelog entry: 

```suggestion
    under the ``cloud_provider_metadata`` option.
```